modules/ROOT/pages/view.adoc

```diff
@@ -3,66 +3,66 @@ ifndef::env-site,env-github[]
-. Click *Save*. 
+. Click *+New*.
+. In the *New view* dialog, enter a *Name* for the view, and click *Save*.
+. Configure the view by selecting which environments, xref:use-tags-in-visualizer.adoc[tags], and services to include.
+. Click *Save*.
 
 You can also create a new view by selecting *Save as* from the drop-down menu to save your current filter selection updates as a new view and preserve the original view.
 
 [NOTE]
 When you save a view, it saves both your *Production* and *Sandbox* configurations.
 
-Once you save the view, it becomes accessible to other users, meaning that any user who has access to the apps in the environment sees the view in their list of views. 
+Once you save the view, it becomes accessible to other users, meaning that any user who has access to the apps in the environment sees the view in their list of views.
 
 == Edit a View
 
-Users can make changes to their private view's filters, and save them. However, only users with the *Visualizer Editor* permission can edit defined views. 
+Users can make changes to their private view's filters, and save them. However, only users with the *Visualizer Editor* permission can edit defined views.
 
 . Make the desired changes to the current filters. +
-An asterisk appears next to the view name until you click *Save* in the side panel. 
-. Click *Save*. 
-. (Visualizer editor only) Click *Save as* to save your updates as a new view and preserve the original view. 
+An asterisk appears next to the view name until you click *Save* in the *View* card.
+. Click *Save*.
+. (Visualizer editor only) Click *Save as* to save your updates as a new view and preserve the original view.
 +
 image::visualizer-save-as-new-view.png[]
 
 == Rename a View
 
-Users can rename their private view, but only users with *Visualizer Editor* permission can rename defined views. 
+Users can rename their private view, but only users with *Visualizer Editor* permission can rename defined views.
 
-. Click the drop-down arrow next to *Save*, and select *Rename*. 
+. Click the drop-down arrow next to *Save*, and click *Rename*.
 +
 image::visualizer-rename-view.png[]
-. In the *Rename view* dialog, enter a new *Name*, and click *Save*. 
+. In the *Rename view* dialog, enter a new *Name*, and click *Save*.
 
 == Share a View
 
-Sharing a defined view is automatic in the sense that when a new view is created and saved, it is visible in the view list of users who have access to the environment and apps in the view. 
+Sharing a defined view is automatic in the sense that when a new view is created and saved, it is visible in the view list of users who have access to the environment and apps in the view.
 
-You can also click *Copy* next to the shareable link and send it to the user you want to share the view with. 
+You can also click *Copy Sharing URL* in the list next to *Save* and send it to the user you want to share the view with.
```

Comment:
Maybe something like "from the dropdown next to the save button"

Reply:
went with "from the dropdown next to **Save** ...."